### ISSUE TITLE
Comment

### ISSUE THREAD

**Issue by JoeSquared777** [North-West-Wind/islandr.io#19]
I'm trying to help create surviv.io back but I can't really code in words. IF there is anything I could do to help, that would be great!

P.S. you and hsanger should both work on one project, it would be alot easier and faster to get done.

**Comment by alany08** [comment]
Yeah, but as of now we are working on basic optimization and getting the system to work. A PS to your PS: These two projects are fundamentally very different. While resurviv is taking the client and directly copy and pasting it, they will already have a framework at the disadvantage of not understanding it (hopefully soon to understand). This limits them until they get a full understanding of >30,000 lines of obfuscated code, which I can say is very confusing. Our approach at opensurviv is to take the idea of the game, and implement it in our own way, meaning we are designing our own client as well. This means we will be slower to reach original surviv, but when we expand beyond surviv it'll be much easier.

TL;DR:
We have 2 very different approaches and I think it would be good to keep both.

**Comment by Viswas-Programs** [comment]
Yo @JoeSquared777 You join our discord server bro?
Join our discord server here : http://opensurviv.run.place/discord

**Comment by JoeSquared777** [comment]
Don't have discord/its blocked.
